src/Symfony/Component/HttpKernel/Profiler/Profiler.php

```diff
@@ -23,6 +23,8 @@
  * Profiler.
  *
  * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @final since Symfony 4.4
```

Comment:
should be on method `collect()` instead, otherwise we completly close extensibily on this class